core/src/test/scala/org/apache/spark/deploy/master/MasterSuite.scala

```diff
@@ -327,22 +328,31 @@ class MasterSuite extends SparkFunSuite
     val masterUrl = s"http://localhost:${localCluster.masterWebUIPort}"
     try {
       eventually(timeout(5.seconds), interval(100.milliseconds)) {
-        val json = Source.fromURL(s"$masterUrl/json").getLines().mkString("\n")
+        val json = Utils.tryWithResource(
+          Source.fromURL(s"$masterUrl/json")) { source =>
+          source.getLines().mkString("\n")
```

Comment:
nit: Indent (and perhaps `_.getLines...` too?)

Reply:
thx ~ done